docs/settingUp.md

```diff
@@ -59,6 +59,7 @@ More information can be found at [this documentation](https://help.github.com/ar
    * `gradle.properties`<br>
       If you want to use a JDK other than the one specified in your PATH variable, add the value to the variable `org.gradle.java.home`.
       This value must be a valid **JDK 1.7** directory.
+      **Windows users** must be careful while entering in the address. Use a **forward slash(/)** instead of the windows default backward slash(/) while entering the path to the directory. This confusion usually occurs when users copy and paste the path from Windows explorer to gradle.properties. 
```

Comment:
What about 
```markdown
**Windows users** should use **forward slash(/)** instead of the default backward slash(/).
```

Think its not necessary to talk about when this error occurs,.